pkgs/development/python-modules/tappy/default.nix

```
@@ -42,6 +43,6 @@ buildPythonPackage rec {
     changelog = "https://tappy.readthedocs.io/en/latest/releases.html";
     mainProgram = "tappy";
     license = licenses.bsd2;
-    maintainers = with maintainers; [ sfrijters ];
+    maintainers = with maintainers; [ ];
```

```suggestion
    license = lib.licenses.bsd2;
    maintainers = with lib.maintainers; [ ];
```

please remove `with lib;`, https://github.com/NixOS/nixpkgs/issues/292468#issuecomment-2016454242